IRMA/1.0.2/Dockerfile

```diff
@@ -0,0 +1,77 @@
+# FROM defines the base docker image. This command has to come first in the file
+# The 'as' keyword lets you name the folowing stage. We use `app` for the production image
+FROM ubuntu:focal as app
+
+# ARG sets environment variables during the build stage
+# USHER ver is in the conda version style.
+ARG IRMA_VER="1.0.2"
+ARG LABEL_VER="0.6.1"
+
+# LABEL instructions tag the image with metadata that might be important to the user
+# Optional, but highly recommended
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="IRMA"
+LABEL software.version=$IRMA_VER
+LABEL description="IRMA was designed for the robust assembly, variant calling, and phasing of highly variable RNA viruses. Currently IRMA is deployed with modules for influenza, ebolavirus and coronavirus."
+LABEL website="https://wonder.cdc.gov/amd/flu/irma/"
+LABEL license="https://wonder.cdc.gov/amd/flu/irma/disclaimer.html"
+LABEL maintainer="John Arnn"
+LABEL maintainer.email="jarnn@utah.gov"
+
+# RUN executes code during the build
+# Install dependencies via apt-get or yum if using a centos or fedora base
+ENV DEBIAN_FRONTEND=noninteractive
```

Comment:
I know some of the dockerfiles set this as an ENV variable, but it'd be better to change it to ARG so it disappears once the container is built

Reply:
agreed